test/multifile/synthesized-accessors/two-modules/library.swift

```diff
@@ -1,5 +1,7 @@
 // RUN: true
 
+import Foundation
```

Comment:
We want to make tests cross platform where possible. Can you split off a new test if you want to test NSObject subclasses?